From dabd09b141c6c64c73f2c3fd73767a3cc6c5a95e Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 11 Jun 2018 17:50:27 +0530
Subject: [PATCH 0372/1851] dt-bindings: display: xlnx: Add VTC bridge support
 to PL display

commit a4783bb74653ca103048f014bda8b79ff94d1605 from
https://github.com/Xilinx/linux-xlnx.git

Update documentation showing how to add vtc bridge node to PL display
driver device tree node. It is an optional property.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,pl-disp.txt        | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,pl-disp.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,pl-disp.txt
index 232a8a583976..c6034bffc64a 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,pl-disp.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,pl-disp.txt
@@ -20,6 +20,10 @@ Required properties:
   Documentation/devicetree/bindings/graph.txt.
 - reg: Base address and size of device
 
+Optional properties:
+ - xlnx,bridge: bridge phandle
+   This handle is required only when VTC is connected as bridge.
+
 Example:
 
 	drm-pl-disp-drv {
@@ -27,6 +31,7 @@ Example:
 		dmas = <&axi_vdma_0 0>;
 		dma-names = "dma0";
 		xlnx,vformat = "YUYV";
+		xlnx,bridge = <&v_tc_0>;
 		pl_disp_port@0 {
 			reg = <0>;
 			endpoint {
-- 
2.31.1

